static/css/frame.scss

```diff
@@ -47,6 +47,61 @@
   }
 }
 
+.promo-panel {
+  .edit {
+    right: 50px;
+    top: 75px;
```

Comment:
Until we're using this panel in multiple places, it doesn't make sense to abstract out positioning for the edit button. This is confusing as written, since the 'edit' and 'promo-box' classes are applied to the same element. Just get rid of this .edit class and keep all the positioning declarations in one spot.